common/djangoapps/enrollment/tests/test_views.py

```diff
@@ -47,14 +47,15 @@ def assert_enrollment_status(
             self,
             course_id=None,
             username=None,
+            email=None,
             expected_status=status.HTTP_200_OK,
             email_opt_in=None,
             as_server=False,
             mode=CourseMode.DEFAULT_MODE_SLUG,
             is_active=None,
             enrollment_attributes=None,
             min_mongo_calls=0,
-            max_mongo_calls=0,
+            max_mongo_calls=1,  # see student.models.py::1235
```

Comment:
What if the row number changes

Reply:
maybe more informative reference would be in place

Reply:
deleted in https://github.com/raccoongang/edx-platform/pull/2069